docs/source/aws/examples/lambdaerrorsnotify.rst

```diff
@@ -14,7 +14,7 @@ For the notify action in the policy to work you must have setup the c7n_mailer t
 https://github.com/cloud-custodian/cloud-custodian/tree/master/tools/c7n_mailer
 
 Mailer Setup Guide:
-https://devops4solutions.com/cloud-custodian-configure-email/
+https://github.com/cloud-custodian/cloud-custodian/blob/master/tools/c7n_mailer/README.md
```

Comment:
Wondering if we might want to point to the rendered docs link instead of the Markdown source here?

https://cloudcustodian.io/docs/tools/c7n-mailer.html